code/modules/power/lighting/light.dm

```diff
@@ -325,15 +325,19 @@
 		replacer.ReplaceLight(src, user)
 		return
 
-	//SKYRAT EDIT ADDITION
-	if(istype(tool, /obj/item/multitool) && constant_flickering)
-		to_chat(user, span_notice("You start repairing the ballast of [src] with [tool]."))
-		if(do_after(user, 2 SECONDS, src))
-			stop_flickering()
-			to_chat(user, span_notice("You repair the ballast of [src]!"))
-		return
+	//SKYRAT EDIT
+	if(istype(tool, /obj/item/multitool))
+		if(constant_flickering)
+			to_chat(user, span_notice("You start repairing the ballast of [src] with [tool]."))
+			if(do_after(user, 2 SECONDS, src))
+				stop_flickering()
+				to_chat(user, span_notice("You repair the ballast of [src]!"))
+			return
```

Comment:
redundant return if you early returned early as well as moving back the below return one indentation.